sdk/src/ingredient.rs

```diff
@@ -85,19 +85,36 @@ pub struct Ingredient {
 }
 
 impl Ingredient {
-    pub fn new(title: &str, format: &str, instance_id: &str) -> Self {
+    /// Returns an [Ingredient]
```

Comment:
These new docs look great!

One minor request: The Rust API docs typically are punctuated as complete sentences. (See, for example, https://doc.rust-lang.org/std/string/struct.String.html#method.new.) Would you please add a `.` at end of the initial doc comment?

Reply:
I update all the doc comments to end with `.`